paddle/fluid/operators/listen_and_serv_op.h

```diff
@@ -68,5 +70,25 @@ class ListenAndServOp : public framework::OperatorBase {
   static std::atomic_int selected_port_;
 };
 
+class SignalHandler {
+ public:
```

Comment:
Please merge the two public sections. For your reference: https://google.github.io/styleguide/cppguide.html#Declaration_Order